src/pages/SingleTweet.tsx

```diff
@@ -0,0 +1,224 @@
+import { Dropdown } from "react-bootstrap";
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  max-width: 598px;
+  padding: 0 16px;
+  text-align: left;
+`;
+const Author = styled.div`
+  height: 48px;
+  margin-top: 12px;
```

Comment:
Use bootstrap for margins and paddings. [bootstrap-spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/)

Reply:
done